docs/tutorials/scroll.md

```diff
@@ -0,0 +1,130 @@
+---
+section: docs
+layout: docs
+title: Scroll Snap
+---
+
+These examples shows the various scroll snap properties supported by atomic.
+
+### `scroll-behavior` construct
+
+The scroll-behavior CSS property sets the behavior for a scrolling box when scrolling is triggered by the navigation, `CSSOM` scrolling APIs or `SCROLL-SNAP`.
```

Comment:
I like linking to the MDN article for the user so they can link off to the docs pretty quickly. Let's do that for the other mentions below.

```suggestion
The [scroll-behavior](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior) CSS property sets the behavior for a scrolling box when scrolling is triggered by the navigation, `CSSOM` scrolling APIs or `SCROLL-SNAP`.
```

Why is `SCROLL-SNAP` in uppercase?